Directions Example/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleSignature</key>
```

Comment:
Looks like `agvtool new-marketing-version 0.33.0` also needs to be run on the repository. The line above here still says `0.31.0`.